app/views/home/landing.html.erb

```diff
@@ -104,7 +104,7 @@
             return handler();
           }
           $defaultPillBox.hide();
-          draw(window[collection].slice(0,10), collection);
+          draw(window[collection], collection);
```

Comment:
We should try to move this JS script out of the view as soon as possible, preferably in this pull request

Reply:
I don't think it should be on this PR. The JS is very complicated and will most likely break things when moving and refactoring it. I would say to let this JS be here for this PR and create another PR specifically for that. What do you think?